cmake/lint.cmake

```diff
@@ -21,6 +22,9 @@ function(lint)
   endif()
 endfunction()
 
+# Find bundled programs
+list(INSERT CMAKE_PREFIX_PATH 0 ${DEPS_PREFIX})
```

Comment:
This is another reason why I was hesitant about moving the detection to a separate module: we don't use `find_program` for any bundled (`.deps`) things outside of the top-level CMakeLists.txt.

Unfortunately this PR doesn't seem to fix the issue for me...

Reply:
Why does that matter? Meh, I can move the find_program back to the root CMakeLists then.

What issue do you have? What does the text say? It works for me.